### ISSUE TITLE
Suggestions for the publication

### ISSUE THREAD

**Issue by srmnitc** [SMTG-Bham/easyunfold#40]
This is part of the JOSS review over at https://github.com/openjournals/joss-reviews/issues/5974

- Line 10-11, 14 could be strengthened by a citation.
- Line 36, k-points were not mentioned before, the theory comes only later. Probably a one line introduction could be really useful. 
- Line 47, VASP and CASTEP needs citations.

**Comment by zhubonan** [comment]
Thanks for the comments. I have updated the paper accordingly. 

> Line 10-11, 14 could be strengthened by a citation.

I did not find a suitable citation for describing how to run band structure calculation and its wide use - in most cases, the procedures are described in the documentation of each DFT codes. Nevertheless, I included a citation of a widely used helper tool (at least, in my group..), [sumo](https://smtg-bham.github.io/sumo/index.html), to run band structure calculations and plot the results.

**Comment by zhubonan** [comment]
> Line 36, k-points were not mentioned before, the theory comes only later. Probably a one line introduction could be really useful.

Good point, I think it is not necessary to introduce the concept of k-points here. The relevant sentence has been updated to: 

https://github.com/SMTG-Bham/easyunfold/blob/1ae2b7546e74d9656fee82ae20a759c5ad84d64c/paper/paper.md?plain=1#L61-L62
